Tests/TestAppUITests/TestAppUITests.swift

```diff
@@ -15,6 +15,11 @@ class TestAppUITests: XCTestCase {
         XCTAssertTrue(screen.isLoaded)
     }
 
+    func testIsLoadedReturnsTrueWhenScreenWithMultipleElementsIsLoaded() throws {
+        let screen = try MultipleElementsScreen()
+        XCTAssertTrue(screen.isLoaded)
+    }
```

Comment:
Similarly to the test a few lines above, this simply creates a screen that matches the one loaded in the app and checks that `isLoaded` returns `true`.

The only difference is that `MultipleElementScreen` uses two expected element getter closures.